internal/rest/k8s_utils.go

```diff
@@ -143,9 +143,10 @@ func (l *leader) SyncObjectStatuses() {
 	}
 
 	publisher := status.NewStatusPublisher()
```

Comment:
@swathinsankaran/ @abhishekbsingh  : As per discussion we need set leader always true in wcp env.